koku/api/resource_types/openshift_projects/view.py

```diff
@@ -47,8 +47,14 @@ def list(self, request):
         if request.user.admin:
             return super().list(request)
         elif request.user.access:
-            user_access = request.user.access.get("openshift.project", {}).get("read", [])
+            if request.user.access.get("openshift.cluster", {}).get("read", []):
+                user_access = request.user.access.get("openshift.cluster", {}).get("read", [])
+                # We hold a copy of the filtered queryset just incase the user has a wildcard for user access
+                query_holder = self.queryset.filter(cluster_id__in=user_access)
+            elif request.user.access.get("openshift.project", {}).get("read", []):
+                user_access = request.user.access.get("openshift.project", {}).get("read", [])
+                query_holder = self.queryset.filter(cluster_id__in=user_access)
```

Comment:
Is this correct? Shouldn't it be:
```suggestion
                query_holder = self.queryset.filter(namespace__in=user_access)
```